planner/core/logical_plan_builder.go

```diff
@@ -852,7 +860,7 @@ func (b *PlanBuilder) buildUnion(union *ast.UnionStmt) (LogicalPlan, error) {
 	oldLen := unionPlan.Schema().Len()
 
 	if union.OrderBy != nil {
-		unionPlan, err = b.buildSort(unionPlan, union.OrderBy.Items, nil)
+		unionPlan, err = b.buildSort(unionPlan, union.OrderBy.Items, nil, nil)
```

Comment:
Just a question: here we disable using window function in sort of UNION ALL right?

Reply:
Yes.